source/base/utilities.cc

```diff
@@ -319,6 +318,15 @@ namespace Utilities
   }
 
 
+  std::vector<std::string>
+  split_string_list (const std::string &s,
+                     const char delimiter)
+  {
+    std::string d = ",";
+    d[0] = delimiter;
```

Comment:
Not wrong, but couldn't you have just written this as `std::string((char[]){delimiter, '\0'})`?